ui/src/components/diagram/WorkflowGraph.jsx

```diff
@@ -338,8 +338,21 @@ class WorkflowGraph extends React.Component {
     inner.selectAll("g.node").on("click", this.handleClick);
   };
 
+  /**
+   * Get the taskRef id base on browsers
+   * @param e
+   * @returns {string | undefined} The id of the task ref
+   */
+  getTaskRef = (e) => {
+    const isFirefox = navigator.userAgent.toLowerCase().indexOf("firefox") > -1;
+    if (isFirefox) {
+      return e.target?.parentNode?.id;
+    }
+    return e?.path[1]?.id || e?.path[2]?.id; // could be 2 layers down
+  };
+
   handleClick = (e) => {
-    const taskRef = e.path[1].id || e.path[2].id; // could be 2 layers down
+    const taskRef = this.getTaskRef(e);
```

Comment:
I believe this could be made simpler with `event.composedPath`. Chrome recommends updating to this for the now deprecated `event.path` API https://chromestatus.com/feature/5726124632965120.

It also seems to be compatible with all major browsers https://developer.mozilla.org/en-US/docs/Web/API/Event/composedPath#browser_compatibility

Reply:
added the fix here https://github.com/Netflix/conductor/pull/3459